src/sentry/workflow_engine/endpoints/validators/base/detector.py

```diff
@@ -26,6 +26,7 @@ class BaseDetectorTypeValidator(CamelSnakeSerializer):
         help_text="Name of the uptime monitor",
     )
     detector_type = serializers.CharField()
+    config = serializers.JSONField(default={})
```

Comment:
are you thinking we should expose this to the API so the UI can also validate against it?

Reply:
So for now, this is just so that the tests pass. `config` ends up not being included in `validated_data`, so it's just a workaround for now.

I'm not sure if config should be exposed, but that's how the tests are written at the moment. We can figure out if we want to let people pass configs, or just allow params to be passed specific to a given validator.